mobify/platform-scaffold
# [Integration Manager] Convert all cart fetches to the command.

The Merlin's connector logic for cart fetches was duplicated in the old location as a transitional measure. This PR removes it as preparation for the cart schema.

 **JIRA**: preparatory for https://mobify.atlassian.net/browse/WEBDATA-10

## Changes
- Remove the duplicate parser and action
- Change all of the references to point to the connector.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to the cart and see that it shows up in the mini cart and main cart page.
